docs/getting_started.md

```diff
@@ -120,8 +120,8 @@ kubectl describe pod busybox-curl-7bd5767999-spf7v
     Host Port: 0/TCP
     Args:
       start
-      --uri
-      core.openfeature.dev/default/featureflagconfiguration-sample
+      --sources
+      - '[{"uri":"core.openfeature.dev/default/featureflagconfiguration-sample","provider":"kubernetes"}]'
```

Comment:
we can drop the core.openfeature.dev prefix in the uri here as (I believe) they are no longer required by any sync implementations